src/i_joystick.c

```diff
@@ -44,6 +39,12 @@ static SDL_Joystick *joystick = NULL;
 
 static int usejoystick = 0;
 
+// Use analog joystick input
+int joystick_analog = 0;
```

Comment:
These are not static because we need to know in `g_game.c` if analog input is enabled or not. If  there's a better way to expose that I can refactor it.